knackload/test/test_load.py

```diff
@@ -9,6 +9,8 @@
 import requests
 import knackload
 
+KNACK_OBJECT_ID = 1000
```

Comment:
KNACK_OBJECT_ID is an integer in the test but a string in the settings files. Please settle on one!